protocol/features.json

```diff
@@ -0,0 +1,45 @@
ETP-007", "0054-NETP-008", "0054-NETP-009", "0054-NETP-010", "0054-NETP-011", "0057-TRAN-063", "0065-FTCO-005", "0065-FTCO-006", "0065-FTCO-007", "0065-FTCO-008", "0066-VALW-004", "0066-VALW-005", "0068-MATC-060", "0068-MATC-061", "0068-MATC-062", "0068-MATC-063", "0068-MATC-064", "0068-MATC-065", "0068-MATC-066", "0068-MATC-067", "0068-MATC-068", "0068-MATC-069", "0068-MATC-070", "0068-MATC-071", "0068-MATC-072", "0068-MATC-073", "0068-MATC-074", "0068-MATC-075", "0068-MATC-076", "0068-MATC-077", "0068-MATC-078", "0068-MATC-079", "0068-MATC-080", "0068-MATC-081", "0068-MATC-082", "0068-MATC-083", "0068-MATC-084", "0068-MATC-085", "0068-MATC-086", "0068-MATC-087", "0068-MATC-088", "0068-MATC-089", "0068-MATC-090", "0068-MATC-091", "0068-MATC-092", "0069-VCBS-089", "0069-VCBS-090", "0069-VCBS-091", "0069-VCBS-092", "0069-VCBS-093", "0069-VCBS-094", "0069-VCBS-095", "0069-VCBS-096", "0069-VCBS-097", "0069-VCBS-098", "0069-VCBS-099", "0069-VCBS-100", "0069-VCBS-101", "0069-VCBS-102", "0069-VCBS-103", "0069-VCBS-104", "0069-VCBS-105", "0069-VCBS-106", "0069-VCBS-107", "0069-VCBS-108", "0069-VCBS-109", "0069-VCBS-110", "0069-VCBS-111", "0069-VCBS-112", "0069-VCBS-113", "0069-VCBS-114", "0069-VCBS-115", "0069-VCBS-116", "0069-VCBS-117", "0069-VCBS-118", "0069-VCBS-119", "0069-VCBS-120", "0069-VCBS-121", "0069-VCBS-122", "0069-VCBS-123", "0069-VCBS-124", "0069-VCBS-125", "0069-VCBS-126", "0069-VCBS-127", "0069-VCBS-128", "0069-VCBS-129", "0069-VCBS-130", "0069-VCBS-131", "0069-VCBS-132", "0069-VCBS-133", "0069-VCBS-134", "0069-VCBS-135", "0069-VCBS-136", "0069-VCBS-137", "0069-VCBS-138", "0069-VCBS-139", "0069-VCBS-140", "0069-VCBS-141", "0069-VCBS-142", "0069-VCBS-143", "0069-VCBS-144", "0069-VCBS-145", "0069-VCBS-146", "0069-VCBS-147", "0069-VCBS-148", "0069-VCBS-149", "0069-VCBS-150", "0069-VCBS-151", "0069-VCBS-152", "0069-VCBS-153", "0069-VCBS-154", "0069-VCBS-155", "0069-VCBS-156", "0069-VCBS-157", "0069-VCBS-158", "0069-VCBS-159", "0069-VCBS-160", "0069-VCBS-161", "0069-VCBS-162", "0069-VCBS-163", "0069-VCBS-164", "0069-VCBS-165", "0069-VCBS-166", "0069-VCBS-167", "0069-VCBS-168", "0069-VCBS-169", "0069-VCBS-170", "0069-VCBS-171", "0069-VCBS-172", "0069-VCBS-173", "0069-VCBS-174", "0069-VCBS-175", "0069-VCBS-176", "0069-VCBS-177", "0070-MKTD-008", "0070-MKTD-009", "0070-MKTD-010", "0070-MKTD-011", "0070-MKTD-012", "0070-MKTD-013", "0070-MKTD-014", "0070-MKTD-015", "0073-LIMN-077", "0073-LIMN-078", "0073-LIMN-080", "0073-LIMN-082", "0073-LIMN-083", "0073-LIMN-084", "0073-LIMN-086", "0073-LIMN-087", "0073-LIMN-090", "0073-LIMN-091", "0073-LIMN-092", "0073-LIMN-093", "0073-LIMN-094", "0073-LIMN-095", "0073-LIMN-096", "0073-LIMN-097", "0073-LIMN-098", "0073-LIMN-099", "0073-LIMN-100", "0073-LIMN-101", "0073-LIMN-102", "0073-LIMN-103", "0073-LIMN-104", "0074-BTCH-011", "0074-BTCH-012", "0074-BTCH-013", "0074-BTCH-014", "0074-BTCH-015", "0074-BTCH-016", "0074-BTCH-017", "0074-BTCH-018", "0074-BTCH-019", "0074-BTCH-020", "0079-TGAP-006", "0079-TGAP-007"
```

Comment:
I dont think it should matter but should this be on a new line?

Reply:
It doesn’t matter but it should yeah